spread.yaml

```diff
@@ -37,10 +37,7 @@ environment:
     MANAGED_DEVICE: "false"
     # a global setting for LXD channel to use in the tests
     LXD_SNAP_CHANNEL: "latest/edge"
-    # TODO: as of 2021-11-18 ubuntu-image in candidate and up has a bug which is
-    # fixed in https://github.com/canonical/ubuntu-image/pull/26, revert to
-    # latest/candidate when the fix lands in master and the snap is released
-    UBUNTU_IMAGE_SNAP_CHANNEL: "latest/stable"
+    UBUNTU_IMAGE_SNAP_CHANNEL: "latest/edge"
```

Comment:
do we even need this env var anymore?

Reply:
actually it's needed again :/ (at least for 16.04)

Reply:
ack makes sense